### PR TITLE
[VEN-2253] FDUSD market configuration

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR introduces a VIP to add FDUSD market on `bsc testnet` using chaos labs [recommendations](https://chaos-labs.notion.site/EXT-Chaos-Labs-Parmeter-Recommendations-for-FDUSD-Listing-On-Venus-b8779c8d34aa4df2a3b80e11fe531dee)

Resolves VEN-2253

### CODE REVIEW

**Comment by GitGuru7** [comment]
testnet VIP LGTM

File: simulations/vip-221/vip-221-testnet/simulations.ts
Comment (GitGuru7) [inline simulations/vip-221/vip-221-testnet/simulations.ts:1]: I think we can use Vtoken check (src/vip-framework/checks/checkVToken.ts).

**Review by chechu — COMMENTED** [review]
Testnet VIP lgtm

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/222?chainId=56

**Pull request merged**